deploy/rbac-clowdapp.yml

```diff
@@ -69,6 +69,11 @@ objects:
             cpu: 200m
             memory: 256Mi
         env:
+          - name: SENTRY_DSN
```

Comment:
@bennyturns is there a good way to verify that the SENTRY_DSN value for stage will be correct when it's deployed? The secret is set in vault; I simply don't know how to test this part without actually deploying to the stage environment.